### DISCUSSION TITLE
Does Middle support regular USB mice too?

### DISCUSSION THREAD

**Discussion by TurtleDynasty** [rxhanson/Middle-Localization#56] [Q&A]
I am a newish mac user. Until now, I never needed to use one long enough to notice that regular USB mice can't middle click on Macs. Does this software work with regular USB mice too? I have a feeling this would be a very common question. Could this be added to FAQ?

**Comment by rxhanson** [comment]
Middle does not allow physical mouse button remapping - it’s just for Magic Mouse and Apple trackpads. My other app,[Multitouch ](https://multitouch.app) , does allow physical mouse button remapping, and is free to test out for 30 days to see if it works with your mouse. 

The default behavior of macOS is to treat a mouse with a physical third button as the middle button. If you aren’t experiencing this, that typically means you have a nonstandard mouse (seemingly pretty uncommon) or it is intended to be used with software provided by the manufacturer.
